k8s/aat/common/cnp/plum-recipe-backend.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: plum-recipe-backend
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: plum-recipe-backend
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
```

Comment:
needs to be hmctspublic to work on aat, may require publishing a new chart version